### DISCUSSION TITLE
Help to set default terminal

### DISCUSSION THREAD

**Discussion by CaioMS2000** [eclipse-theia/theia#13792] [Q&A]
I'm trying to set PowerShell as default terminal, so everytime i click "new terminal" it opens using powershell, i thought i should just set this configuration but it keeps opening using CMD
![image](https://github.com/eclipse-theia/theia/assets/72468013/1f24e415-69e6-4d19-91fb-926d82af4193)
i've set the same configuration(C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe) in "user" and "workspace" section

**Comment by dartains** [comment]
same issue, not able to change cmd by pws

**Comment by msujew** [comment]
Hey @CaioMS2000, @dartains,

sorry for the late response, I had to find time to investigate what's wrong. With the latest refactoring of the terminal profiles, things changed a bit. You now need to set the following preference in your settings.json:

```json
"terminal.integrated.profiles.windows": {
    "cmd": {
        "path": "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
    }
}
```

To open that settings.json, you can navigate to the `terminal.integrated.profiles.windows` preference (search for it in the UI) and click on the `Edit in setting.json` link:

![grafik](https://github.com/user-attachments/assets/b0d4b8e1-621f-4de6-b85c-8f15c652c941)

I've created https://github.com/eclipse-theia/theia/issues/14609 to track UX issues related to the profile selection.

**Reply by dceccon87** [reply]
In Theia 1.68.201 version it work  (Windows 11)
